app/models/stacks_image.rb

```diff
@@ -41,9 +66,10 @@ def druid
   end
 
   def path
-    path = super
-
-    path + '.jp2' if path
+    @path ||= begin
+                pth = PathService.for(druid, file_name)
+                pth + '.jp2' if pth
```

Comment:
Does this `jp2` suffix logic seem worth pushing into `PathService`?

Reply:
I was thinking about that for a few moments., but tried to keep focused on other matters.  Is it important?